core/domain/exp_services.py

```diff
@@ -949,6 +1033,8 @@ def update_exploration(
             feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX)
 
     updated_exploration = apply_change_list(exploration_id, change_list)
+    if bool(get_story_id_linked_to_exploration(exploration_id)):
```

Comment:
Please remove bool conversion or compare against `None` or `''` directly, converting to bool is not helpful in if statements.

Reply:
Done